vivisect/vamp/msvc/__init__.py

```diff
@@ -1,32 +1,78 @@
 """
-envi.bytesig signature stuff for Microsoft Visual Studio
+envi.bytesig signatures for Microsoft Visual Studio
```

Comment:
i know the pain of maintaining delta's you're using regularly, and the PR... but the vamp sigs in this PR don't help it. :)